### DISCUSSION TITLE
2010/11/08/%E7%BB%88%E4%BA%8E%E6%90%AC%E5%88%B0wordpress%E4%B8%8A%E6%9D%A5%E4%BA%86

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#300] [Announcements]
# 2010/11/08/%E7%BB%88%E4%BA%8E%E6%90%AC%E5%88%B0wordpress%E4%B8%8A%E6%9D%A5%E4%BA%86

前一阵子一直在犹豫要不要搬到这里，主要是怕这个网址将来被和谐。后来想想，还是先不管了，不搬家就没法发新贴了。就算和谐了，翻墙也不算难事。

https://blog.qizhen.xyz/2010/11/08/%E7%BB%88%E4%BA%8E%E6%90%AC%E5%88%B0wordpress%E4%B8%8A%E6%9D%A5%E4%BA%86

**Comment by ruanqizhen** [comment]
之前博客下的留言：

ruanqizhen
2010年11月12日 09:20 编辑
你说的有道理，改回 “我和LabVIEW” :)

回复
foolooo
2010年11月12日 08:54 编辑
阮老师终于搬家完事了。恭喜。
我估计被墙是早晚的，但是没办法，目前就这个稳定些。

另 “我 以及 LabVIEW”没有“我和LabVIEW”上口啊～

回复
